Python/Machine Learning/Disneyland Review Analysis/README.md

```diff
@@ -0,0 +1,97 @@
+# Disneyland Review Analysis
+
+### Aim:
+
+The aim of this project is to analyse the reviews given by visitors from different countries of the world using Natural Language Processing (NLP) to understand the sentiment of the reviews and classify them using Sentiment Analysis metrics like Sentiment Polarity and VADER Polarity. This processed data is then fed to different classifier models (DecisionTreeClassifier and RandomForestClassifier) to get trained and predict the sentiment of the test reviews.
+
+### Install the dependencies:
+- Install Python 3.8 and above Kit.
+- Further install these libraries using terminal:
+
+```
+    pip install pandas
+```
+
+```
+    pip install numpy
+```
+
+```
+    pip install matplotlib
+```
+
+```
+    pip install nltk
+```
+
+```
+    pip install wordcloud
+```
+
+```
+    pip install pillow
+```
+
+### Quick Start
+- Clone this repository.
+```
+git clone https://github.com/abhijeet007rocks8/Dev-Scripts.git
+```
+- Change directory.
+```
+cd Python
+```
+
+```
+cd Machine Learning
+```
+
+```
+cd Disneyland Review Analysis
+```
+
+### Libraries Used:
+- pandas
+- numpy
+- seaborn
+- matplotlib
+- nltk
+- wordcloud
+- scikit-learn
+- string
+- PIL
+
+### Topics Covered:
+- Loading the dataset
+- Dealing with missing values
+- Pre-processing of data
+- Data visualization (via graphs and maps)
+- Word cloud
+- Label encoding on various columns
+- Performing sentiment analysis using SentimentIntensityAnalyzer and VADER Polarity
+- Splitting the data
+- Using Tf-IDF Vectorizer
+- Using Decision Tree Classifier
+- Using Random Forest Classifier
+
+### Code Walkthrough
+![Code walkthrough of disneyland review analysis](./Images/ezgif-2-a5e660b421.mp4)
```

Comment:
this image is not working and you need to include screen recorded video.

follow all the rules which are mentioned in [Rules.md](https://github.com/abhijeet007rocks8/Dev-Scripts/blob/main/Rules.md) . **Adding video is mandatory** till then I can't merge this PR.

How to upload video in markdown then watch this video : 
[https://drive.google.com/file/d/12cjBwDaTCK4pu7cqNI_FNuRAZwamCywP/view?usp=sharing](https://drive.google.com/file/d/12cjBwDaTCK4pu7cqNI_FNuRAZwamCywP/view?usp=sharing&authuser=0)

**Note** : Screen Recorded video should include 
- code go-through 
- what are the files you have included
- run your project in screen recording